xml/System.Resources.Extensions/PreserializedResourceWriter.xml

```diff
@@ -345,9 +410,16 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <summary>Writes all resources to the output stream in the system default format.</summary>
```

Comment:
We should probably say something different than `system default format` since it isn't always written in the system default format.  If the resources added to the writer can be represented in the system default format understood by ResourcesReader, they will be written as such.  If the resources require runtime serialization other than that supported by the system default format (BinaryFormatter) then we will write them using the format understood by DeserializingResourceReader.

Reply:
So the summary can be:

```suggestion
        <summary>Writes all resources to the output stream.</summary>
```

And I can extend the remark below. How does that sound, @ericstj?

Reply:
Yeah